content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -0,0 +1,559 @@
+1. Enable the OpenTelemetry Collector and configure the essential ports:
+   {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
+datadog:
+  ...
+  otelCollector:
+    enabled: true
+    ports:
+      - containerPort: "4317" # default port for OpenTelemetry gRPC receiver.
+        hostPort: "4317"
+        name: otel-grpc
+      - containerPort: "4318" # default port for OpenTelemetry HTTP receiver
+        hostPort: "4318"
+        name: otel-http
+   {{< /code-block >}}
+1. (Optional) Enable additional Datadog features.
+   <div class="alert alert-danger">Enabling these features may incur additional charges. Review the <a href="https://www.datadoghq.com/pricing/"> pricing page</a> and talk to your CSM before proceeding.</div>
+   {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
+datadog:
+  ...
+  apm:
+    portEnabled: true
+    peer_tags_aggregation: true
+    compute_stats_by_span_kind: true
+    peer_service_aggregation: true
+  orchestratorExplorer:
+    enabled: true
+  processAgent:
+    enabled: true
+    processCollection: true
+  networkMonitoring:
+    enabled: true
+   {{< /code-block >}}
+   
+{{% collapse-content title="Completed datadog-values.yaml file" level="p" %}}
+Your `datadog-values.yaml` file should look something like this:
+{{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="false" >}}
+agents:
+  image:
+    repository: datadog/agent-dev
+    tag: nightly-ot-beta-main-jmx
+    doNotCheckTag: true
+
+datadog:
+  apiKeyExistingSecret: datadog-secret
+  appKeyExistingSecret: datadog-secret
+
+  otelCollector:
+    enabled: true
+    ports:
+      - containerPort: "4317"
+        hostPort: "4317"
+        name: otel-grpc
+      - containerPort: "4318"
+        hostPort: "4318"
+        name: otel-http
+
+  podLabelsAsTags:
+    app: kube_app
+    release: helm_release
+   {{< /code-block >}}
+
+{{% /collapse-content %}}
+   
+### Configure the OpenTelemetry Collector
+
+The DataDog Helm chart provides a sample OpenTelemetry Collector configuration that's a great starting point for most projects. This section walks you through the predefined pipelines and included OpenTelemetry components.
+
+This is the full OpenTelemetry Collector configuration in `otel-config.yaml`:
+
+{{< code-block lang="yaml" filename="otel-config.yaml" disable_copy="false" collapsible="true" >}}
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "otelcol"
+          scrape_interval: 10s
+          static_configs:
+            - targets: ["0.0.0.0:8888"]
+  otlp:
+    protocols:
+      grpc:
+         endpoint: 0.0.0.0:{{ include "get-port-number-from-name" (dict "ports" .Values.datadog.otelCollector.ports "portName" "otel-grpc") }}
```

Comment:
> In Configure the OpenTelemetry Collector, this line is included in `otel-config.yaml`: `endpoint: 0.0.0.0:{{ include "get-port-number-from-name" (dict "ports" .Values.datadog.otelCollector.ports "portName" "otel-grpc") }}`. Are we expecting that this dynamically populates? It doesn't seem to work.

These will be dynamically replaced by Helm when running `helm install`/`helm upgrade`. 
Helm chart is the only supported way to install the agent with otel collector at the moment, so we expect customers to be familiar with Helm. 

@brett0000FF if you think it still might be confusing, lets simplify it:

```suggestion
         endpoint: 0.0.0.0: 4317
```

Reply:
Ah, I think this is fine to leave then. We use a very similar syntax to create dynamic content in the docs, so that tripped me up. Thanks!